plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -264,7 +267,7 @@ public static SliceReadFunction charReadFunction(CharType charType)
 
     public static SliceWriteFunction charWriteFunction()
     {
-        return (statement, index, value) -> statement.setString(index, value.toStringUtf8());
+        return SliceWriteFunction.of(Types.VARCHAR, (statement, index, value) -> statement.setString(index, value.toStringUtf8()));
```

Comment:
Why not CHAR? Why not NCHAR?

Reply:
nchar is not recognized and accepted by Postgres: `Caused by: org.postgresql.util.PSQLException: Unknown Types value`

Reply:
> Why not CHAR?